### PR TITLE
[SDPSUP-3771] Update Drupal Version

### PR DESCRIPTION
**Pull request by edyuenyw**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPSUP-3771

### Changed

1.  Changed Privacy Statement text to Drupal 9
2.  Updated hook

### Screenshots
FE
To see the change: https://app.pr-901.vic-gov-au.sdp2.sdp.vic.gov.au/contact-us
image

BE
image

### CODE REVIEW

File: tide_webform.install [hunk 1]
```diff
@@ -220,3 +220,34 @@ function tide_webform_update_8007() {
     $block->save();
   }
 }
+
+/**
+ * Updating service version.
+ */
+function tide_webform_update_8008() {
+  if (\Drupal::moduleHandler()->moduleExists('tide_webform')) {
```
Comment (vincent-gao) [inline tide_webform.install:228]: hey  @edyuenyw  doesn't need to check whether tide_webform exists or not.

File: tide_webform.install [hunk 2]
```diff
@@ -220,3 +220,34 @@ function tide_webform_update_8007() {
...
+    $config->set('privacy_statement.content', $content_value);
+    $config->save();
+  }
+}
+
+/**
+ * Updating service version on existing webform.
+ */
+function tide_webform_update_8009() {
+  if (\Drupal::moduleHandler()->moduleExists('webform')) {
```
Comment (vincent-gao) [inline tide_webform.install:242]: hey  @edyuenyw  doesn't need to check whether `webform` exists or not.

File: tide_webform.install (hunk 1 above)
Comment (edyuenyw) [reply tide_webform.install:228]: Thank you @vincent-gao. Will remove it.

File: tide_webform.install (hunk 2 above)
Comment (edyuenyw) [reply tide_webform.install:242]: Thank you @vincent-gao. Will remove it.

**Review by vincent-gao — APPROVED** [review]

**Pull request merged**